src/mgr/DaemonServer.cc

```diff
@@ -2057,9 +2058,15 @@ bool DaemonServer::_handle_command(
 	    }
 	    d += to_string(i);
 	  }
+	  char wear_level_str[16] = {0};
+	  if (dev.wear_level >= 0) {
+	    snprintf(wear_level_str, sizeof(wear_level_str)-1, "%d%%",
+		     (int)(100.1 * dev.wear_level));
```

Comment:
```suggestion
            wear_level_str = fmt::format("{:.2f}", 100.1 * dev.wear_level);
```

where 2 digits are displayed after the decimal point. so "3.1415926" is printed like "3.14".

Reply:
The original data source is an integer percentage, so there's not point in showing more precision here.